### ISSUE TITLE
Disable token revocation property is not working in APIM 4.1.0

### ISSUE THREAD

**Issue by ug205** [wso2/api-manager#1691]
### Description

The following disable token revocation property is not working in APIM 4.1.0
```
[oauth.jwt.renew_token_without_revoking_existing]
enable = true
```

### Steps to Reproduce

1. Add the property

```
[oauth.jwt.renew_token_without_revoking_existing]
enable = true
```

2. Start the Server
3. Generate an access token and invoke API 
4. Generate another token before the initial one is expired
5. Reinvoke with the initial token and observe that is not valid anymore.

### Affected Component

APIM

### Version

4.1.0

**Comment by YasasRangika** [comment]
need to merge the public fix: https://github.com/wso2/product-apim/pull/13200

**Comment by tharikaGitHub** [comment]
**Public fixes for this issue are the following.**

1. https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2053
2. https://github.com/wso2/carbon-identity-framework/pull/4535
3. https://github.com/wso2/product-apim/pull/13200
4. https://github.com/wso2/carbon-apimgt/pull/12238

But the carbon-identity-framework fixes are not ported to the 5.25.x branch. Hence reopening.

**Comment by tharikaGitHub** [comment]
Closing as the PR [1] is merged.

[1] https://github.com/wso2/carbon-identity-framework/pull/5615
